Description: enable rkyv default features for tests.
 Add a dev-dependency on rkyv with default features are enabled, so the
 default rkyv size is selected during tests.

 We do not want to enable default features in the regular dependency as
 applications should be free to choose what rkyv size they use.

Author: Peter Michael Green <plugwash@debian.org>

--- rust-num-complex-0.4.6.orig/Cargo.toml
+++ rust-num-complex-0.4.6/Cargo.toml
@@ -70,6 +70,9 @@ version = "0.7"
 optional = true
 default-features = false
 
+[dev-dependencies.rkyv]
+version = "0.7"
+
 [dependencies.serde]
 version = "1.0"
 optional = true
